src/sagemaker/clarify.py

```diff
@@ -1307,56 +1290,176 @@ def run_explainability(
                   the Trial Component will be unassociated.
                 * ``'TrialComponentDisplayName'`` is used for display in Amazon SageMaker Studio.
         """  # noqa E501  # pylint: disable=c0301
-        analysis_config = data_config.get_config()
-        predictor_config = model_config.get_predictor_config()
-        if isinstance(model_scores, ModelPredictedLabelConfig):
-            (
-                probability_threshold,
-                predicted_label_config,
-            ) = model_scores.get_predictor_config()
+        analysis_config = _AnalysisConfigGenerator.explainability(
+            data_config,
+            model_config,
+            model_scores,
+            explainability_config
+        )
+        # when name is either not provided (is None) or an empty string ("")
+        job_name = job_name or utils.name_from_base(self.job_name_prefix or "Clarify-Explainability")
+        return self._run(
+            data_config,
+            analysis_config,
+            wait,
+            logs,
+            job_name,
+            kms_key,
+            experiment_config,
+        )
+
+    def run_bias_and_explainability(self):
```

Comment:
Can we see if we can override the existing run() method for this purpose?

Reply:
`Signature of method 'SageMakerClarifyProcessor.run()' does not match signature of the base method in class 'Processor'`